README.md

```
@@ -1,74 +1,717 @@
+const bugSpotter = await BugSpotter.init({
+  endpoint: 'https://api.bugspotter.com/api/v1/reports',
+  auth: {
+    type: 'api-key',
+    apiKey: 'bgs_your_api_key',
+    projectId: 'your-project-uuid',
+  },
+  showWidget: true,
 });
+```
+
+**CommonJS (Node.js)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)